### DISCUSSION TITLE
Can I add a readme for a slightly more complex use?

### DISCUSSION THREAD

**Discussion by UchiMaki615** [DarkaOnLine/L5-Swagger#579] [General]
The following is just an example.
In my actual development, when I am deploying different projects for each sub-domain within the same API server (repository),
I had a requirement to change the output document for each subdomain.

I had to look at the source code myself to figure out the process... and then I was able to achieve the above.
I don't think the above usage is that uncommon,
If you like, you can expand the readme and make this repository easier to use,
I hope it will make this repository easier to use and more accessible to people.

If this is not a problem, I would like to know how to proceed.
Thank you in advance.

**Comment by DarkaOnLine** [comment]
We have [wiki](https://github.com/DarkaOnLine/L5-Swagger/wiki) for examples and use cases. Please feel free to contribute

**Comment by UchiMaki615** [comment]
@DarkaOnLine 
Thank you for your prompt reply.
We have reviewed the contents and will include them here.
